index.xml

```diff
@@ -14892,6 +15172,11 @@ d.type = et;
           <prod nt='ExtendedAttributeIdentList'>identifier "=" "(" IdentifierList ")"</prod>
           <prod nt='ExtendedAttributeNamedArgList'>identifier "=" identifier "(" ArgumentList ")"</prod>
 
+          <prod nt='Namespace'>"namespace" identifier "{" NamespaceMembers "}" ";"</prod>
+
+          <prod nt='NamespaceMembers'>ExtendedAttributeList NamespaceMember NamespaceMembers | ε</prod>
+          <prod nt='NamespaceMember'>ReturnType OperationRest</prod>
```

Comment:
Well, nobody needs them yet. We can add them later, but in general constants are avoided these days (in favor of simply using well-known strings). I went for minimal for now.